src/file-1.xoo

```diff
@@ -0,0 +1,17 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonarcloud-github-static-resources/v1/common/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:46079/organizations/org0/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:46079/project/issues?id=sample&pullRequest=6987&issues=AWleLu28Eo9FcZJAz3eH&open=AWleLu28Eo9FcZJAz3eH)

[ISSUE_KEY]: AWleLu28Eo9FcZJAz3eH